gtsfm/loader/argoverse_dataset_loader.py

```diff
@@ -30,14 +30,18 @@ def __init__(
         max_num_imgs: int = 20,
         max_lookahead_sec: float = 2,
         camera_name: str = "ring_front_center",
+        max_resolution: int = 760,
     ) -> None:
         """Select the image paths and their corresponding timestamps for images to feed to GTSFM.
         Args:
             dataset_dir: directory where raw Argoverse logs are stored on disk
             log_id: unique ID of vehicle log
             stride: sampling rate, e.g. every 2 images, every 4 images, etc.
             max_num_imgs: number of images to load (starting from beginning of log sequence)
+            max_resolution: integer representing maximum length of image's short side
```

Comment:
maybe replace this docstring to mean that max is applied. This explains what size will be treated as the resolution of the image, but does not indicate we will scale the image.

Maybe something like 'integer representing the maximum length of the smaller of the height/width of the image. If the image resolution max(height, weight) is greater than the max_resolution, it will be downsampled to match the max_resolution'.

Reply:
Thanks, will do.

Reply:
thanks, updated it